### PR TITLE
NEPT-1060: Implement new buttons design.

### PR DESCRIPTION
**Pull request by Fefaine**
Implement new buttons design

### CODE REVIEW

File: profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module [hunk 1]
```diff
@@ -227,9 +233,19 @@ function nexteuropa_fields_formatters_field_formatter_view($entity_type, $entity
...
 
-    case 'ne_bs_button':
-      $modifier = 'btn';
+    case 'ne_button':
+      $modifier = 'ecl-button';
+      $additional_classes = ['arrow-up',
+        'arrow-down',
+        'caret-up',
+        'caret-right',
+      ];
```
Comment (imanoleguskiza) [inline profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module]: caret-down and caret-left are missing

File: profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module [hunk 2]
```diff
@@ -79,21 +79,27 @@ function nexteuropa_fields_formatters_field_formatter_settings_form($field, $ins
...
           'secondary' => 'Secondary',
+          'form' => 'Form',
+          'light' => 'Light',
+          'arrow-up' => 'Arrow Up',
+          'arrow-down' => 'Arrow Down',
+          'carret-up' => 'Carret Up',
+          'carret-down' => 'Carret Down',
+          'carret-left' => 'Carret Left',
+          'carret-right' => 'Carret Right',
         ],
```
Comment (imanoleguskiza) [inline profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module]: there is a typo in the values and the descriptions, Carret should be caret with one single r

File: profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module [hunk 3]
```diff
@@ -93,10 +90,10 @@ function nexteuropa_fields_formatters_field_formatter_settings_form($field, $ins
           'light' => 'Light',
```
Comment (gillesdeudon) [inline profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module]: `Light`, `Block` and `info` do not exist in the ecl button variant. 

So is it normal to find them in the variation options?

File: profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.module [hunk 4]
```diff
@@ -10,6 +10,13 @@ include_once 'nexteuropa_metatags.tokens.inc';
  * Implements hook_metatag_info().
  */
 function nexteuropa_metatags_metatag_info() {
+  $info['group']['tags'] = array(
```
Comment (voidtek) [inline profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.module]: @Fefaine: Can you please answer that question?

Comment (drupol) [inline profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.module]: @voidtek She will be back on Tuesday. If you think that this is unrelated changes, please, update the PR so we do not lose too much time.

**Comment by voidtek** [comment]
:+1:

File: profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.module (hunk 4 above)
Comment (Fefaine) [inline profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.module]: I had to add this at some point when we had the issue with the metatags, since errors were generated.

File: profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module [hunk 5]
```diff
@@ -15,10 +15,7 @@ function nexteuropa_fields_formatters_field_formatter_info() {
       'description' => t('Displays as a button.'),
       'field types' => [
         'link_field',
-        'list_text',
         'taxonomy_term_reference',
```
Comment (gillesdeudon) [inline profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module]: The formatter can be used with "Term reference" fields but why it cannot be used for "Entity reference" fields?

The module is available in the platform and is enabled by default.

Furthermore, it has also a link display like for term reference, via its "Label" formatter

File: profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module [hunk 6]
```diff
@@ -372,10 +367,7 @@ function nexteuropa_fields_formatters_field_formatter_view($entity_type, $entity
             ),
           ),
         );
-        $markup = render($link);
-        $elements[$delta] = array(
-          '#markup' => $markup,
-        );
+        $elements[$delta] = $link;
```
Comment (drupol) [inline profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module]: 👍

File: profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module [hunk 7]
```diff
@@ -323,12 +321,9 @@ function nexteuropa_fields_formatters_field_formatter_view($entity_type, $entity
                 ),
               ),
             );
-            $markup = render($link);
             break;
         }
-        $elements[$delta] = array(
-          '#markup' => $markup,
-        );
+        $elements[$delta] = $link;
```
Comment (drupol) [inline profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module]: 👍

File: profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module [hunk 8]
```diff
@@ -437,12 +429,7 @@ function nexteuropa_fields_formatters_field_formatter_view($entity_type, $entity
               ),
             ),
           );
-
-          $markup = render($link);
-
-          $elements[$delta] = array(
-            '#markup' => $markup,
-          );
+          $elements[$delta] = $link;
```
Comment (drupol) [inline profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module]: 👍  Thx delphine

File: profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module (hunk 5 above)
Comment (Fefaine) [inline profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module]: done

File: profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module (hunk 6 above)
Comment (Fefaine) [inline profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module]: done

**Comment by gillesdeudon** [comment]
👍

**Comment by imanoleguskiza** [comment]
👍
